naptime/src/main/scala/org/coursera/naptime/actions/RestAction.scala

```diff
@@ -231,7 +232,7 @@ trait RestAction[RACType, AuthType, BodyType, KeyType, ResourceType, ResponseTyp
    */
   override def tagRequest(request: RequestHeader): RequestHeader = {
     tags.map { tags =>
-      request.copy(tags = tags)
+      request.addAttr(RequestAttrKey.Tags, tags)
```

Comment:
can you explain this change?

Reply:
sure, so  the copy function is deprecated in play 2.6

https://github.com/playframework/playframework/blob/master/framework/src/play/src/main/scala/play/api/mvc/RequestHeader.scala#L281

I am just copying the internal impl out to avoid the copy deprecation warning

https://github.com/playframework/playframework/blob/master/framework/src/play/src/main/scala/play/api/mvc/RequestHeader.scala#L302